web-components/src/components/card/Card.ts

```diff
@@ -1,53 +1,153 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
 import reset from "@/wc_scss/reset.scss";
 import { customElementWithCheck } from "@/mixins/CustomElementCheck";
-import { html, LitElement, property } from "lit-element";
+import { html, internalProperty, LitElement, property } from "lit-element";
 import styles from "./scss/module.scss";
 import "@/components/icon/Icon";
 import "@/components/button/Button";
-import { ifDefined } from "lit-html/directives/if-defined";
+import "@/components/list/List";
+import "@/components/list/ListItem";
+import "@/components/favorite/Favorite";
+import { repeat } from "lit-html/directives/repeat";
+import { classMap } from "lit-html/directives/class-map";
+import { nothing } from "lit-html";
 
 export namespace Card {
   @customElementWithCheck("md-card")
   export class ELEMENT extends LitElement {
     @property({ type: String }) title = "title";
     @property({ type: String }) subtitle = "";
-    @property({ type: String }) src = "";
-    @property({ type: String }) link = "";
-    @property({ type: String }) linkTitle = "";
-    @property({ type: String }) buttonTitle = "";
+    @property({ type: String }) id = "";
+    @property({ type: Boolean }) fullscreen = false;
+    @property({ type: String }) info = "";
+    @property({ type: Array }) menuOption: (string | { [key: string]: string })[] = [];
```

Comment:
menuOptions - since it is an array it has to be plural

Reply:
Done